controllers/example_controller/keyboard_controller.py

```diff
@@ -0,0 +1,72 @@
+from sr.robot import Robot
+from controller import Keyboard
+
+TIMESTEP = 16
```

Comment:
We must be able to derive this from somewhere?

Reply:
Maybe? Bear in mind that this seems to be expected to be different to the basic timestemp. Perhaps @JennyLynnFletcher can provide more context?

Reply:
This is simply how often the control input is sampled.

Reply:
d0e9162